lndclient/lightning_client.go

```diff
@@ -384,3 +393,36 @@ func (s *lightningClient) ListTransactions(ctx context.Context) ([]*wire.MsgTx,
 
 	return txs, nil
 }
+
+// ChannelBackup retrieves the backup for a particular channel. The backup is
+// returned as an encrypted chanbackup.Single payload.
+func (s *lightningClient) ChannelBackup(ctx context.Context,
+	channelPoint wire.OutPoint) ([]byte, error) {
+
+	req := &lnrpc.ExportChannelBackupRequest{
+		ChanPoint: &lnrpc.ChannelPoint{
+			FundingTxid: &lnrpc.ChannelPoint_FundingTxidBytes{
+				FundingTxidBytes: channelPoint.Hash[:],
+			},
+			OutputIndex: channelPoint.Index,
+		},
+	}
+	resp, err := s.client.ExportChannelBackup(ctx, req)
```

Comment:
In the other calls, we created a per-call timeout context to time out just this request.

Reply:
Good catch, fixed.